plugins/enterkey/plugin.js

```diff
@@ -521,7 +521,8 @@
 		// Check path block specialities:
 		// 1. Cannot be a un-splittable element, e.g. table caption;
 		var path = editor.elementPath();
-		if ( !path.isContextFor( 'p' ) ) {
+
+		if ( path && !path.isContextFor( 'p' ) ) {
```

Comment:
There is similar issue #357 and @Comandeer [mentioned a more generic approach there](https://github.com/ckeditor/ckeditor-dev/pull/357#pullrequestreview-38671902) which may cover all those cases, which is generally a good approach. The cause of the issue is the fact that enter key press is handled while there is no selection (while editable area is focused). This is however a responsibility of the core to handle such situations and `enterkey` plugin by itself also should do some additional checkings as above.

Having said that, I think this check is sufficient for this case.